run_pylint.sh

```diff
@@ -24,19 +24,20 @@ fi
 
 TMPFILENAME="/tmp/run_pylint_$RANDOM"
 PEP8=`which pep8`
-PEP8IGNORE='E731'
+# for readability, allow multiple spaces after commas, colons & before '='
+PEP8IGNORE='E731,E221,E241'
 MSGFMT='(pylint) {msg_id}:{line:3d},{column}: {obj}: {msg}'
 # Disable 'line too long' - will be picked up by pep8
 # Check "note" (W0511) separately
-DISABLEMSG="I0011,R0801,R0904,R0921,R0922,C0301,W0511${SPECIALONE}"
+DISABLEMSG="I0011,R0801,R0904,R0921,R0922,C0301,C0326,W0511${SPECIALONE}"
 INIT_HOOK="
 AP = os.environ.get('AUTOTEST_PATH', '/usr/local/autotest')
 sys.path.append(os.path.abspath(AP + '/..'))
 sys.path.append(os.path.abspath('.'))
 import autotest
 import autotest.common
 "
-SUBTESTDISABLEMSG="I0011,R0801,R0904,E1101,E1002,R0903,F0401,C0103,C0111,W0232,C0301,W0511,${SPECIALONE}"
+SUBTESTDISABLEMSG="I0011,R0801,R0904,E1101,E1002,R0903,F0401,C0103,C0326,C0111,W0232,C0301,W0511,${SPECIALONE}"
```

Comment:
The C0326's are pylint messages, no?  Doing those inline while globally ignoring the one from pep8 seems nonsensical.  I have three-quarters of a mind to say let's just kill pep8 entirely.